tracer/src/Datadog.Trace/Debugger/Snapshots/DebuggerSnapshotSerializer.NotCapturedReason.cs

```diff
@@ -14,7 +14,8 @@ private enum NotCapturedReason
             collectionSize,
             depth,
             fieldCount,
-            timeout
+            timeout,
+            redacted
```

Comment:
Non blocking comment - per latest update in the RFC, we should change this to "redactedByType" and "redactedByIndentifier" so that our frontend will be able to give a more helpful message to the user based on the scenario.

Reply:
Done in b3c60786fcc78c01e87daeec97c11b4d1103b0ef.